system/modules/timelog/partials/templates/timelogwidget.tpl.php

```diff
@@ -82,11 +113,11 @@ function saveTimer() {
                     success: function(data) {
                         var object_data = JSON.parse(data);
                         
-                        start_time = (new Date().getTime() / 1000);
+                        start_time = object_data.start_time ? object_data.start_time : (new Date().getTime() / 1000);
```

Comment:
In Javascript (correct me if I'm wrong), you can't do object accessors like this in a conditional. It will result in an error that may stop execution. Similarly, you can't do this in PHP either, e.g:
`$object_data->start_time ? ...` will throw an error if start_time is not a property of $object_data.

In JS, you should use `object_data.start_time !== undefined ? ... `